### PR TITLE
Refactoring and a little bit styling

### PR DESCRIPTION
**Pull request by MaxMer25**

### CODE REVIEW

File: pages/home/tasks.js [hunk 1]
```diff
@@ -71,24 +75,30 @@ export default function Home() {
...
                   <h2>{task.title}</h2>
-                  <StyledImage
+                  <Image
                     priority={true}
                     width={175}
                     height={122}
                     src={task.image}
                     alt="picture of a task"
-                  ></StyledImage>
+                  ></Image>
```
Comment (stefankorder) [inline pages/home/tasks.js]: You could call the Image component as a self-closing tag like this: `<Image />`

File: pages/home/tasks.js [hunk 2]
```diff
@@ -119,19 +129,26 @@ export default function Home() {
...
+                  <h2>{task.title} </h2>
+                  <Image
                     className="reviewImage el"
                     priority={true}
                     width={175}
                     height={122}
                     src={task.image}
                     alt="picture of a task"
-                  ></StyledImage>
+                  ></Image>
```
Comment (stefankorder) [inline pages/home/tasks.js]: Same as above.

File: pages/home/tasks.js [hunk 3]
```diff
@@ -99,7 +109,7 @@ export default function Home() {
                         change: {review: "in review", whoDid: user.name},
                       })
                     }
-                    className="taskButtons"
+                    className="taskButtons done"
```
Comment (stefankorder) [inline pages/home/tasks.js:112]: Do you mix styled components with css files?

File: pages/home/tasks.js [hunk 4]
```diff
@@ -155,23 +172,31 @@ export default function Home() {
...
                   <h2>{task.title}</h2>
-                  <StyledImage
+                  <Image
                     priority={true}
                     width={175}
                     height={122}
                     src={task.image}
                     alt="picture of a task"
-                  ></StyledImage>
+                  ></Image>
```
Comment (stefankorder) [inline pages/home/tasks.js]: Same as above.

File: pages/home/tasks.js [hunk 5]
```diff
@@ -155,23 +172,31 @@ export default function Home() {
...
+
         {user.type === "Parent" &&
           tasks.map(task => {
             if (task.review !== "in review" && task.review !== "reviewed") {
               return (
                 <StyledListElements key={task._id}>
                   <h2>{task.title}</h2>
-                  <StyledImage
+                  <Image
                     priority={true}
```
Comment (stefankorder) [inline pages/home/tasks.js]: You could also write it like this:

```suggestion
                    priority
```

File: pages/home/tasks.js [hunk 6]
```diff
@@ -119,19 +129,26 @@ export default function Home() {
                   <div className="ex">
                     <Exclamation />
                   </div>
-                  <h2>{task.title}</h2>
-                  <StyledImage
+
+                  <h2>{task.title} </h2>
+                  <Image
                     className="reviewImage el"
                     priority={true}
```
Comment (stefankorder) [inline pages/home/tasks.js]:
```suggestion
                    priority
```

File: pages/home/tasks.js [hunk 7]
```diff
@@ -71,24 +75,30 @@ export default function Home() {
...
         {user.type === "Child" &&
           tasks.map(task => {
-            if (task.review !== "in review") {
+            if (task.review !== "in review" && task.review !== "reviewed") {
               return (
                 <StyledListElements key={task._id}>
                   <h2>{task.title}</h2>
-                  <StyledImage
+                  <Image
                     priority={true}
```
Comment (stefankorder) [inline pages/home/tasks.js]:
```suggestion
                    priority
```

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Take a look at my comments.

File: pages/home/tasks.js (hunk 3 above)
Comment (MaxMer25) [reply pages/home/tasks.js:112]: Hey Stefan, no i defined that class to give this specific buttons a grid area. the class selector for that is in my styled component

**Pull request merged**